pylon/src/nginx.conf.template

```diff
@@ -76,11 +76,11 @@ http {
       subs_filter_types *;
       subs_filter
         \"containerLog\":\"http://([^/]*):8042/(.*)\"
```

Comment:
>8042 [](start = 42, length = 4)

in the future, we should change this hardcode port setting

Reply:
Exactly. Noted.